sql/src/test/java/io/crate/integrationtests/SubSelectIntegrationTest.java

```diff
@@ -65,6 +65,13 @@ public void testSubSelectWhere() throws Exception {
                 "where name like 'Arthur'");
         assertThat(printedTable(response.rows()),
             is("4| Arthur\n"));
+
+        execute("select id, name " +
+                "from (select * from characters where female = true) as ch " +
```

Comment:
please don't add a ilike test into a test case for something else. This makes changing tests later on more difficult as it will be more unclear what the test was supposed to test

We can use the QueryTester framework to very that the built Lucene query behave the way we want.